app/assets/js/rtMedia.backbone.js

```diff
@@ -2098,7 +2212,6 @@ function renderUploadercomment_media( widget_id, parent_id_type ) {
 			rtMediaHook.call( 'rtmedia_js_after_files_added', [ upl, rfiles ] );
 
 			if ( typeof rtmedia_direct_upload_enabled != 'undefined' && rtmedia_direct_upload_enabled == '1' ) {
```

Comment:
@yahilmadakiya
Thanks for pointing this out.
Will make the change and let you know with the same